guillotina/tests/fixtures.py

```diff
@@ -278,10 +298,11 @@ def __init__(self, request):
 
 
 @pytest.fixture(scope='function')
-def guillotina_main(loop):
+def guillotina_main(loop, request):
     globalregistry.reset()
+    test_settings = get_test_settings(request.node)
     aioapp = loop.run_until_complete(
-        make_app(settings=get_db_settings(), loop=loop))
+        make_app(settings=test_settings, loop=loop))
```

Comment:
we still need this to work with `get_db_settings`.

Can you pass `request.node` to `get_db_settings` instead?

I think we need the same for `get_dummy_settings`